app/views/reviews/_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<div>
+  <%= simple_form_for [book, review] do |form_builder| %>
+    <%= form_builder.input :body, label: "Your review", required: true %>
```

Comment:
Good point, though I do prefer to open the form and clearly see the required fields? Happy to go either way on this